backend/python/app/services/implementations/story_service.py

```diff
@@ -229,6 +229,77 @@ def get_story_translations(
             self.logger.error(str(error))
             raise error
 
+    def get_story_translation_tests(
+        self,
+        user,
+        language=None,
+        level=None,
+        stage=None,
+        story_title=None,
+    ):
+        try:
+            filters = [StoryTranslation.is_test]
```

Comment:
neat! Im not familiar with this notation

Reply:
oh it's equivalent to `StoryTranslation.is_test == true` i believe